dev-packages/node-integration-tests/package.json

```diff
@@ -42,6 +42,7 @@
     "cors": "^2.8.5",
     "cron": "^3.1.6",
     "express": "^4.17.3",
+    "fastify": "~4.23.2",
```

Comment:
For future readers - we had to pin this at `~4.23.2` because `4.33+` only works with TS 5, which breaks the rest of our tests 😬